BrainPortal/app/views/userfiles/_quality_control_panel.html.erb

```diff
@@ -36,10 +36,12 @@
     <legend>Navigation: file <%= @current_index + 1 %> out of <%= @filelist.size %></legend>
     <%= link_to_userfile_if_accessible(@userfile, current_user, :html_options => { :target => '_blank' }) %>
     <p>
-    <% if @current_index > 0 %>
+    <% first_file = @current_index > 0 %>
```

Comment:
I don't get it. The variable 'first_file' is TRUE when this is NOT the first file ?!?!? What kind of naming is this?